insonmnia/structs/task.go

```diff
@@ -0,0 +1,37 @@
+package structs
+
+import (
+	"errors"
+
+	"github.com/sonm-io/core/proto"
+)
+
+var (
+	errDealRequired  = errors.New("deal is required")
+	errBidIdRequired = errors.New("bid id must be non-empty")
+)
+
+type StartTaskRequest struct {
+	*sonm.HubStartTaskRequest
```

Comment:
we use pb as sonm alias everywhere. I Iike `sonm.` but I vote for consistency

Reply:
I like `sonm` too.

But it's module local change, it does not affect aliases in other modules. In case of rename-everything a strongly disrespect such changes, but for migration state it's ok.